front/lib/api/assistant/global_agents/configurations/dust/dust.ts

```diff
@@ -112,132 +113,145 @@ export function _getDustGlobalAgent(
     and the public internet before answering the user's question.
 
 4. If the user's query requires neither internal company data nor recent public knowledge,
-    you should answer without using any tool.`;
+    you should answer without using any tool.
+`;
 
-  const requestComplexityPrompt = `<request_complexity>
-  Always start by classifying requests as "simple" or "complex".
-  You must follow the appropriate guidelines for each case.
+  const requestComplexityPrompt = `
+<request_complexity>
+Always start by classifying requests as "simple" or "complex".
+You must follow the appropriate guidelines for each case.
 
-  A request is complex if any of the following conditions are met:
-  - It requires deep exploration of the user's internal company data, understanding the structure of the company data, running several (3+) searches
-  - It requires doing several web searches, or browsing 3+ web pages
-  - It requires running SQL queries
-  - It requires 3+ steps of tool uses
-  - The user specifically asks for a "deep dive", a "deep research", a "comprehensive search", a "comprehensive analysis" or "comprehensive report" or other terms that indicate a deep research task
+A request is complex if any of the following conditions are met:
+- It requires deep exploration of the user's internal company data, understanding the structure of the company data, running several (3+) searches
+- It requires doing several web searches, or browsing 3+ web pages
+- It requires running SQL queries
+- It requires 3+ steps of tool uses
+- The user specifically asks for a "deep dive", a "deep research", a "comprehensive search", a "comprehensive analysis" or "comprehensive report" or other terms that indicate a deep research task
 
-  Any other request is considered "simple".
+Any other request is considered "simple".
 
-  <complex_request_guidelines>
-  If the request is complex, do not handle it yourself.
-  Immediately delegate the request to the deep dive agent by using the \`deep_dive\` tool.
-  </complex_request_guidelines>
+<complex_request_guidelines>
+If the request is complex, do not handle it yourself.
+Immediately delegate the request to the deep dive agent by using the \`deep_dive\` tool.
```

Comment:
😱 this is not the valid tool name! I know not in your PR but since you're going to have to add a commit to stop importing globalAgentGuidelines can you fix this? 🙏🏻

Reply:
It is the right name ? https://github.com/dust-tt/dust/blob/63a1c35ee26eb5e2008848e4c34fc659d3b2432d/front/lib/api/assistant/global_agents/configurations/dust/dust.ts#L398